File: arrays/remove_element/remove_element.ts
export class Solution {
    /**
     * @param {string} testInput
     * @return {any} Returns either boolean to validate the test cases or a custom result.
     */
    run_solution(testInput: any) {
        const [nums, val] = testInput;

        let k = 0;

        nums.forEach(e => {
            if (e !== val) {
                nums[k] = e;
                k++;
            }
        });

        return nums;
    }
}

File: arrays/remove_element/remove_element.py
"""Solution Module"""


class Solution:
    """Solution Class"""

    def run_solution(self, test_input):
        """Solution Method"""
        [arr, val] = test_input
        k = 0
        for i, e in enumerate(arr):
            if e != val:
                arr[k] = arr[i]
                k += 1
        return k
